src/dev/src/app/content-container.component.ts

```diff
@@ -56,5 +58,19 @@ import { APP_ROUTES } from './app.routing';
         `,
 })
 export class AppContentContainerComponent {
+  private static readonly ROOT_BREADCRUMB_ELEMENT = { label: 'Home', url: '/' };
+
   public routes: Route[] = APP_ROUTES;
+  public breadcrumbElements: BreadcrumbModel[] = [AppContentContainerComponent.ROOT_BREADCRUMB_ELEMENT];
+
+  constructor(private router: Router) {
+    router.events.subscribe(val => {
```

Comment:
You are right, but in our application, the breadcrumb should work a little bit different. It shouldn't change on every navigation, only sometimes, when we make it change manually (through a service). So the changes here are only a simple demonstration how it can work.

Reply:
I see, then please add this service to the component.
So you can use the component via the service directly and doesn't have to implement the service on your side.
Then you don't have to make the BreadcrumbModel public, but only the service.